jenkinsci/ldap-plugin
# [JENKINS-73825] ldap allows insecure configurations

See [JENKINS-73825](https://issues.jenkins.io/browse/JENKINS-73825)
In FIPS mode, now LDAP don't allows the insecure configuration. Also it shows the error messages in case of insecure ldap url/ short password

### Testing
Manually testing is done with below scenarios
1. Configure insecure ldap url & error should thrown + form data should not be saved  ✅ 
2. Configure secure ldap url & error should thrown + form data should not be saved  ✅ 
3. In advance configuration manager password less than 14 is not allowed  ✅ 
4. CASC configuration unit test to load the ldap configuration i.e. **CasCFipsTest**  ✅
     * When configuration has secure url + valid password length  
     * When configuration has insecure url/valid password length **CasCFipsTest.configure_ldap_for_invalid**   
5. UI Integration Test ✅ 
     * For Insecure ldap server url
     * For invalid manager password size





### Submitter checklist
